tests/plugins/clipboard_figure.test.ts

```diff
@@ -128,6 +128,24 @@ describe.each(["chart", "image"])("Clipboard for %s figures", (type: string) =>
     expect(model.getters.getFigures(sheetId)).toHaveLength(0);
   });
 
+  test("Figure is copied to edge of the sheet", () => {
+    model.dispatch("UPDATE_FIGURE", {
+      sheetId,
+      id: figureId,
+      height: 256,
+      width: 257,
+    });
+    model.dispatch("SELECT_FIGURE", { id: figureId });
+    copy(model);
+    paste(model, "Z100");
+    const copiedFigure = model.getters.getFigure(sheetId, getCopiedFigureId());
```

Comment:
you can force the non-null assertion here rather than calling it on every subsequent call
```suggestion
    const copiedFigure = model.getters.getFigure(sheetId, getCopiedFigureId())!;
```